### PR TITLE
Updated some entity fields and class mappings

### PR DESCRIPTION
**Pull request by strikerrocker**

### CODE REVIEW

File: mappings/net/minecraft/entity/mob/IllagerEntity.mapping [hunk 1]
```diff
@@ -1,4 +1,5 @@
 CLASS apb net/minecraft/entity/mob/IllagerEntity
+	CLASS apb$a IllagerState
```
Comment (asiekierka) [inline mappings/net/minecraft/entity/mob/IllagerEntity.mapping]: It's an inner class; why not State?

File: mappings/net/minecraft/entity/mob/SlimeEntity.mapping [hunk 2]
```diff
@@ -17,6 +17,7 @@ CLASS aqe net/minecraft/entity/mob/SlimeEntity
 		METHOD a canStart ()Z
 		METHOD e tick ()V
 	FIELD bD SLIME_SIZE Lpr;
+	FIELD bE wasOnGround Z
```
Comment (asiekierka) [inline mappings/net/minecraft/entity/mob/SlimeEntity.mapping]: When is this true?

**Review by asiekierka — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/entity/mob/SlimeEntity.mapping (hunk 2 above)
Comment (strikerrocker) [reply mappings/net/minecraft/entity/mob/SlimeEntity.mapping]: What do ya mean?

Comment (strikerrocker) [reply mappings/net/minecraft/entity/mob/SlimeEntity.mapping]: Anwyays this is true when onGround was true.

Comment (asiekierka) [reply mappings/net/minecraft/entity/mob/SlimeEntity.mapping]: And it's never set to false, ever?

**Comment by strikerrocker** [comment]
Did the requested changes

**Review by asiekierka — APPROVED** [review]

**Review by SquidDev — APPROVED** [review]
I'm slightly worried by the `isJohnny` field, but who am I to question Mojang?

**Review by shadowfacts — APPROVED** [review]

**Pull request merged**